docs/installation/local.mdx

```diff
@@ -39,23 +68,58 @@ capact environment create kind --wait 5m
 capact environment create k3d --wait 5m
 ```
 
-> **NOTE** You can customize the behavior with additional flags. See the all options in the [`capact environment create k3d`](../cli/commands/capact_environment_create_k3d.md) command documentation.
+:::note
+You can customize the behavior with additional flags. See all the options in the [`capact environment create k3d`](../cli/commands/capact_environment_create_k3d.md) command documentation.
+:::
 
 </TabItem>
 </Tabs>
 
 ## Install Capact
 
-Once you have your local Kubernetes cluster set up, install Capact with the following command:
+Once you have your local Kubernetes cluster set up, install Capact. You can choose from two types of release channels: **Stable** and **Latest**:
+
+- The **Stable** channel refers to the official [Capact releases](https://github.com/capactio/capact/releases).
+- The **Latest** channel refers to the latest successful build from the commit on the [Capact repository `main` branch](https://github.com/capactio/capact/commits/main).
+
+:::tip
+By default all Capact components are installed. You can disable monitoring installation to reduce time and resource consumption.
+
+To install Capact without `monitoring`, add the `--install-component=capact,neo4j,ingress-nginx,argo,cert-manager,kubed` flag to the `capact install` command.
+:::
+
+<Tabs
+	groupId="ref"
+	defaultValue="stable"
+	values={[
+		{label: 'Stable', value: 'stable'},
+		{label: 'Latest', value: 'latest'},
+	]}>
+<TabItem value="stable">
 
 ```bash
 capact install
 ```
 
-> **NOTE** You can customize the behavior with additional flags. See the all options in the [`capact install`](../cli/commands/capact_install.md) command documentation.
+Wait for the command to finish.
+
+</TabItem>
+<TabItem value="latest">
+
```

Comment:
I would explain what "latest" means for us:

```suggestion

To install Capact from the latest commit on `main` branch, run:
```

Reply:
discussed - not valid